packages/fury-adapter-remote/package.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "fury-adapter-remote",
+  "version": "0.0.1",
+  "description": "Provide adapter for Element API based on callin remote API site",
+  "author": "Apiary.io <support@apiary.io>",
+  "license": "MIT",
+  "main": "./lib/adapter.js",
+  "homepage": "https://github.com/apiaryio/api-elements.js/tree/master/packages/fury-adapter-remote",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/apiaryio/api-elements.js.git"
+  },
+  "scripts": {
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "test": "mocha test"
+  },
+  "engines": {
+    "node": ">=4"
```

We're only testing on Node 6 onwards so let's make the range we support what we're testing in CI:

```suggestion
    "node": ">=6"
```

---

will update, BTW on fury-adapter-oas3-parser` https://github.com/apiaryio/api-elements.js/blob/master/packages/fury-adapter-oas3-parser/package.json#L37

Is also `"node": ">=4"` is it correct?

---

It's not, good catch. I can fix in separate PR.